projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/pojo/Analysis.java

```diff
@@ -0,0 +1,78 @@
+package org.batfish.datamodel.pojo;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.batfish.datamodel.questions.Question;
+
+/**
+ * The {@link Analysis Analysis} is an Object representation of the analysis for Batfish service.
+ *
+ * <p>Each {@link Analysis Analysis} contains a name and a mapping from question name to {@link
+ * Question question} for questions in the Analysis.
+ */
+public class Analysis {
+  private static final String PROP_NAME = "name";
+  private static final String PROP_QUESTIONS = "questions";
+
+  private String _name;
+  private @Nullable Map<String, Question> _questions;
```

Comment:
I am not sure here, questions in Analysis should have a name(directory name), but our existing Question type does not has this field. Other thinking: 1. add a string field to Question class, but I think this will break our existing tests because we write the Question to .ref, also the name of the Question is not consistent, it should not work on today's testing strategy. 2. Create another "AnalysisQuestion" type which contains a name and a Question instance. Since it is so similar to Question, I not sure whether it is worth to do it.  Also, is Jackson fine with Maps?